### DISCUSSION TITLE
Total Newby with some questions

### DISCUSSION THREAD

**Discussion by FlyingMoose007** [bigcommerce/catalyst#634] [Q&A] [answered]
Hello, Guys.

First of all thanks in advance for your help and patience. Since I am a total newbie. I have very limited knowledge of coding so Catalyst looks extremely attractive to me

I managed to set up a Catalyst project with the CLI, (I tried to deploy it on Vercel but when I tried to install the BigComerce Integration, It got stuck on "Completing Installation... forever") This is my first problem.

However, my main question is, once I have set up a project and connected my test store. How Do I start to edit the UI of the store? From what I understand it has a fully customizable UI kit, but I cannot understand where I do this.

I played a little bit with the Catalyst storefront in Figma and I really like it, But I have no clue where to design my actual store. 

Again thanks for the help!

**Comment by christensenep** [comment, marked as answer]
Hello!

First, regarding the Vercel deployment getting stuck on the installation step, we'll look into that and get it fixed, sorry you ran into that issue.

As for editing the UI, currently you would do that by editing the code itself. We're working on more extensive documentation of our component architecture and styling to make this easier to get started.

That being said, if you're looking for a no-code, drag-and-drop experience, you can check out our integration with the [Makeswift](https://www.makeswift.com/) visual editor. Check out this [guide](https://docs.makeswift.com/integrations/bigcommerce) if you're interested in going that route.
